assets/tests/abac_tests.yaml

```diff
@@ -530,6 +769,81 @@ tests:
                 relation: viewer
                 user: user:jon
             expectation: true
+        listUsersAssertions:
```

Comment:
For good measure, can we add the ListObjects assertions above as well so that we have a clear indication of how this behavior relates to Check and ListObjects.

Reply:
Done in https://github.com/openfga/openfga/pull/1566